sbncode/DetSim/FilterSimEnergyDeposits_module.cc

```diff
@@ -50,55 +41,38 @@ class FilterSimEnergyDeposits : public art::EDProducer {
   // Declare member data here.
 
   art::InputTag fInitSimEnergyDepositLabel;
-  std::string fOutSimEnergyDepositLabel;
-  double fP1_x;
-  double fP1_y;
-  double fP1_z;
-  double fP2_x;
-  double fP2_y;
-  double fP2_z;
-  static constexpr auto& kModuleName = "FilterSimEnergyDeposit";
-
-  bool isWithinVolume(TVector3 p1, TVector3 p2, TVector3 sedep);
-  double ShiftX(double z);
+  geo::BoxBoundedGeo fBox;
+  static constexpr auto kModuleName = "FilterSimEnergyDeposit";
+  const double ShiftX(double z);
+  double fA;
+  double fB;
+  double fC;
 
 };
 
 
 FilterSimEnergyDeposits::FilterSimEnergyDeposits(fhicl::ParameterSet const& p)
   : EDProducer{p}
   , fInitSimEnergyDepositLabel{p.get<art::InputTag>("InitSimEnergyDepositLabel", "undefined")}
-  , fOutSimEnergyDepositLabel{p.get<std::string>("OutSimEnergyDepositLabel", "undefined")}
-  , fP1_x{p.get<double>("P1_X", 0.0)}
-  , fP1_y{p.get<double>("P1_Y", 0.0)}
-  , fP1_z{p.get<double>("P1_Z", 0.0)}
-  , fP2_x{p.get<double>("P2_X", 0.0)}
-  , fP2_y{p.get<double>("P2_Y", 0.0)}
-  , fP2_z{p.get<double>("P2_Z", 0.0)}
+  , fBox{p.get<double>("P1_X"), p.get<double>("P2_X"),
+	 p.get<double>("P1_Y"), p.get<double>("P2_Y"),
+	 p.get<double>("P1_Z"), p.get<double>("P2_Z")}
+  , fA{p.get<double>("A", 0.0)}
+  , fB{p.get<double>("B", 0.0)}
+  , fC{p.get<double>("C", 0.0)}
```

Comment:
Please remove the default values: with these values the module will not work correctly (divide by 0), and it may give the illusion to the users that no configuration is needed.